static/app/views/replays/details.tsx

```diff
@@ -72,58 +78,103 @@ class ReplayDetails extends AsyncView<Props, State> {
+
+  /* eslint-disable-next-line no-console */
+  console.log({fetchError, fetching, event, replayEvents, rrwebEvents});
+
+  const renderMain = () => {
+    if (fetching) {
+      return <LoadingIndicator />;
+    }
     if (!event) {
       return <NotFound />;
     }
-
-    const eventView = this.getEventView();
-    const projectSlug = event.projectSlug || event['project.name']; // seems janky
-
     return (
-      <NoPaddingContent>
-        <Layout.Header>
-          <Layout.HeaderContent>
-            <Breadcrumbs
-              crumbs={[
-                {
-                  to: `/organizations/${orgSlug}/replays/`,
-                  label: t('Replays'),
-                },
-                {label: t('Replay Details')}, // TODO: put replay ID or something here
-              ]}
-            />
-            <EventHeader event={event} />
-          </Layout.HeaderContent>
-        </Layout.Header>
-
-        <Layout.Body>
-          <Layout.Main>
-            <RRWebIntegration event={event} orgId={orgSlug} projectId={projectSlug} />
-          </Layout.Main>
-          <Layout.Side>
-            <TagsTable
-              generateUrl={this.generateTagUrl}
-              event={event}
-              query={eventView.query}
-            />
-          </Layout.Side>
-        </Layout.Body>
-      </NoPaddingContent>
+      <React.Fragment>
+        <BaseRRWebReplayer events={rrwebEvents} />
+        {replayEvents?.map(replayEvent => (
+          <React.Fragment key={replayEvent.id}>
+            <TitleWrapper>ReplayEvent: {replayEvent.id}</TitleWrapper>
+            {replayEvent.entries.filter(isReplayEventEntity).map(entry => (
+              <EventEntry
+                key={`${replayEvent.id}+${entry.type}`}
+                projectSlug={getProjectSlug(replayEvent)}
+                // group={group}
+                organization={props.organization}
+                event={replayEvent}
+                entry={entry}
+                route={props.route}
+                router={props.router}
+              />
+            ))}
+          </React.Fragment>
+        ))}
+      </React.Fragment>
     );
-  }
+  };
+
+  const renderSide = () => {
+    if (event) {
+      return <TagsTable generateUrl={() => ''} event={event} query="" />;
+    }
+    return null;
+  };
+
+  return (
+    <NoPaddingContent>
+      <Layout.Header>
+        <Layout.HeaderContent>
+          <Breadcrumbs
+            crumbs={[
+              {
+                to: `/organizations/${orgSlug}/replays/`,
+                label: t('Replays'),
+              },
+              {label: t('Replay Details')}, // TODO: put replay ID or something here
+            ]}
+          />
+          {event ? <EventHeader event={event} /> : null}
+        </Layout.HeaderContent>
+      </Layout.Header>
+      <Layout.Body>
+        <Layout.Main>{renderMain()}</Layout.Main>
+        <Layout.Side>{renderSide()}</Layout.Side>
```

Comment:
I know there was a TSC discussion about this but not sure where it landed (I think this is fine)

Reply:
We should revisit it later. I think as the page gets bigger and more flushed out this part of the file is going to be subject to lots of change.